content/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/automating-dependabot-with-github-actions.md

```diff
@@ -263,6 +263,7 @@ jobs:
         run: gh pr merge --auto --merge "$PR_URL"
         env:
           PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Change `GITHUB_TOKEN: ${{  secrets.GITHUB_TOKEN  }}`
to `GITHUB_TOKEN: '${{  secrets.GITHUB_TOKEN   }}'`

Reply:
I believe this is ok without quotes since it is setting an env variable.